config/config.yml.example

```diff
@@ -0,0 +1,22 @@
+# SUSE Rancher Setup - external configuration
+# The commented keys will be loaded into the application configuration.
+# Only documented keys are accepted.
+
+shared:
+
+  # Which engines are loaded for this workflow?
+  # * Order determines menu order
+  # * Engines without UI components should be last
+  engines:
+    - AWS
+    - ShirtSize
+    - RancherOnEks
```

Comment:
Is the k8s version setting for the control plane targeted for a separate PR. Based on this factoring in config I could see a

```
- RancherOnEks:
  - k8s_version: 1.22
```

which can then be nicely carried forward to other CSPs, for example

```
- RancherOnAzure:
  - k8s_version: 1.23
```
Such a config would also help in accommodating version drift of available k8s versions across CSPs.

Reply:
Good call. I can do that as a follow-up

Reply:
I filed #102 for tracking.